forum/settings.py

```diff
@@ -228,3 +229,8 @@ def running_tests():
     # For tests execution, the fastest password hasher is used
     # (may increase execution speed ×10 and more)
     PASSWORD_HASHERS = ("django.contrib.auth.hashers.MD5PasswordHasher",)
+    DELAY_FOR_LOGIN = 5
+    ATTEMPTS_FOR_LOGIN = 2
+else:
+    DELAY_FOR_LOGIN = 600  # delay time for login in seconds
+    ATTEMPTS_FOR_LOGIN = 10  # attempts for login during delay for login
```

Comment:
Its OK for now but as for me this is bad practice to have such conditions in settings. If you want to use settings for unittests you can use @override_settings decorator

Reply:
This decorator works only if we run single testcase. Smth like this: "python manage.py test authentication.tests.test_user_login". If we run all tests at once (python manage.py test) this decorator would't work. I found this in docs: 
"Warning

The settings file contains some settings that are only consulted during initialization of Django internals. If you change them with override_settings, the setting is changed if you access it via the django.conf.settings module, however, Django’s internals access it differently. Effectively, using override_settings() or modify_settings() with these settings is probably not going to do what you expect it to do."

So, when we are running a specific test case  the override decorator takes effect. When we are running a whole suite specific testcase with override decorator probably isn't the first one that's being run and decorator wouldn't work.
The same for modify_settings() decorator. So I think it's not our varient.

Reply:
I'll create a task.